src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryJavaDocCheck.java

```diff
@@ -7,7 +7,7 @@
 class InputIncorrectSummaryJavaDocCheck {
 
     /**
-     * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}
+     * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}.
```

Comment:
please move all updates in unrelated to functional fix in Check, that is caused by better working functionality, to separate commit, with message "supplemental: fix periods in javadoc summary".

Reply:
Done, thanks

Reply:
I still this supplemental update in functional commit. We would appreciate if you move all such in separate commit, to let us focus on functional part as much as possible.

Reply:
if you make separate PR with such commit , we can merge it instantly , example - https://github.com/checkstyle/checkstyle/pull/14722

Reply:
> I still this supplemental update in functional commit. We would appreciate if you move all such in separate commit, to let us focus on functional part as much as possible. there are few more left.

The tests won't pass without this change though, since this is the test input. Isn't this important for making sense of the functional behaviour of the change?

Reply:
I know, that it why I recommend to move all such fixes to separate commit and PR and merge it very quickly, as it is backward compatible.
We do reviews on low energy and small screen, if you help us to minimize PR, your PR will be merged quicker.

Reply:
Okay, thanks for the context. I'll see if I can split it up into two smaller commits, and whether I can raise a separate bug that can be fixed independently.

Reply:
If you see more defects in this Check, better to create new issue and separate PR. The bigger PR the slower it is accepting

Reply:
I've raised some other issues, which these changes fix as well:
- https://github.com/checkstyle/checkstyle/issues/14750
- https://github.com/checkstyle/checkstyle/issues/14751

I'm not sure if I can fix those without also fixing this.

Reply:
extra issue is required only we can do this independently.
if split is not easy, it is ok to let PR be big.